src/main/java/app/model/boundary/BoundaryImp.java

```diff
@@ -4,23 +4,37 @@
 import app.controller.linAlg.Intersection;
 import app.controller.linAlg.Line;
 import app.controller.linAlg.Vector;
+import app.view.simulation.Info;
 import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.paint.Color;
 
 import java.awt.geom.Line2D;
 
 public class BoundaryImp implements Boundary
 {
     protected Vector a;
     protected Vector b;
+    protected Color colour;
 
     public BoundaryImp(Vector a, Vector b)
     {
         this.a = a;
         this.b = b;
+        colour = Color.WHITE;
     }
 
     @Override
-    public void draw(GraphicsContext gc) {}
+    public void draw(GraphicsContext gc)
+    {
+        if(colour == Color.WHITE)
```

Comment:
wouldn't it be better to set it to null than? since it just means that it won't be drawn

Reply:
Yeah, sure